guides/cookbook.md

```diff
@@ -391,6 +391,217 @@ The result of `getBytes()` from a text or binary writer can subsequently
 be passed as the parameter to `makeReader()` in order to read the Ion data.
 </div>
 
+<div class="tabpane Python" markdown="1">
+Ion-Python provides two distict APIs for reading and writing Ion: a non-blocking event-based API, and
+a blocking `dump/load` API called `simpleion`, which is reminiscent of the popular `simplejson`
+JSON processing API. Where applicable, this cookbook will illustrate how to implement the examples using
+both APIs.
+
+The following examples show how Ion data can be read and written.
+
+#### Using simpleion
+
+```python
+from amazon.ion import simpleion
+
+data = u'{hello: \"world\"}'
+value = simpleion.loads(data)
+print(u'hello %s' % (value[u'hello'],))
+```
+
+Result: `hello world`.
+
+To read data from a file-like object instead of a string or unicode object, use `load` instead.
+
+```python
+from io import BytesIO
+
```

Comment:
I purposely left out imports that had been imported in previous examples to avoid excessive verbosity. I'm torn about this because I think that most readers actually *won't* copy/paste complete examples and try to run them (and for those that do, adding a missing import is a trivial task). Adding extra lines to every example can hurt readability and make the examples seem more complex than they actually are.

However, after adding in all of the missing imports to all of the examples and viewing the results... it's probably not that bad, I guess.